### DISCUSSION TITLE
how to filter contact after account was setted

### DISCUSSION THREAD

**Discussion by newminder** [YetiForceCompany/YetiForceCRM#16880] [How to] [answered]
in the sales order module, when I choose quote, the account was auto filled and connected to the quote. then I want contact could be auto filled according to the account, how to make it?  please help advise.

**Comment by mariuszkrzaczkowski** [comment, marked as answer]
https://github.com/YetiForceCompany/YetiForceCRM/blob/dc3b46124752b39bfe9b4688e709f907db052c86/app_data/moduleHierarchy.php#L64

**Reply by newminder** [reply]
thanks very much,  fixed it by your help.

**Reply by newminder** [reply]
question: what if I got 2 contacts after adding the extra codes, then i want to choose the one its related record like %quotation%, how should I do?

**Reply by newminder** [reply]
I wrote 
in SQuotes
		'accountid'=> ['
		    Contacts'=>[
		        'accountid'=>['parentid'],
		        'contactid'=>['contactno'],
			],
		],
but does not work, and no error.
